content/metadata.yaml

```
@@ -65,8 +65,8 @@ authors:
     name: Krutika S. Gaonkar
     initials: KSG
     orcid: 0000-0003-0838-2405
-    twitter: 29aprilkrutika
-    email: gaonkark@chop.edu
+    twitter: @aggokittu
```

No `@` and should be good -

```suggestion
    twitter: aggokittu
```

---

oh shoot